packages/manager-components/src/utils/price.formatter.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { ODS_TEXT_COLOR_HUE, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  OvhSubsidiary,
+  OvhSubsidiaryToCurrency,
+  IntervalUnitType,
+} from '../enumTypes';
+import './translations/translations';
+
+export interface PriceOptions {
+  ovhSubsidiary: OvhSubsidiary;
+  tFunction: (key: string) => string;
+}
+
+const getPrice = (value: number, tax?: number): number => {
+  const valueWithTax = tax ? value + tax : value;
+  return valueWithTax / 100000000;
+};
+
+const convertIntervalPrice = (
+  price: number,
+  intervalUnit: IntervalUnitType,
+): number => {
+  switch (intervalUnit) {
+    case IntervalUnitType.day:
+      return price / 365;
+    case IntervalUnitType.month:
+      return price / 12;
+    case IntervalUnitType.year:
+      return price;
+    case IntervalUnitType.none:
+      return price;
+    default:
+      return price;
+  }
+};
+
+const getPriceTextFormatted = (
+  ovhSubsidiary: OvhSubsidiary,
+  priceValue: number,
+): string => {
+  const currencyCode = OvhSubsidiaryToCurrency[ovhSubsidiary] || 'EUR';
+  return new Intl.NumberFormat(undefined, {
```

Comment:
A locale must be provided otherwise only the English format will be applied for any locales
Please create a map like `OvhSubsidiaryToCurrency` to handle this